sap/cli/dataelement.py

```
@@ -26,6 +29,135 @@ def instance(self, connection, name, args, metadata=None):
+        """
+        try:
+            domain = sap.adt.Domain(connection, domain_name.upper())
+            domain.fetch()
+
+            console.printout(f'    Description: {domain.description}')
+            console.printout('')
+
+            # Use shared formatter with indentation to align under Type section
+            sap.cli.domain_formatter.format_domain_human(console, domain, indent='    ')
+
+        except Exception as ex:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)